source/Playnite/Localization/LocSource.xaml

```diff
@@ -372,8 +372,8 @@ New database location will only be used after application is restarted.</sys:Str
     <sys:String x:Key="LOCSteamBackgroundSourceScreenshot">Store Screenshot</sys:String>
     <sys:String x:Key="LOCSteamBackgroundSourceStore">Store Background</sys:String>
     <sys:String x:Key="LOCBackgroundImageScreenOptionTooltip" xml:space="preserve">Doesn't apply retrospectively to existing games without re-downloading metadata.</sys:String>
-    <sys:String x:Key="LOCSettingsForceDownloadPlaynite">Force sync play time</sys:String>
-    <sys:String x:Key="LOCSettingsForceDownloadPlayniteTooltip">Always download recorded play time from remote library during library sync, ignoring time recorded by Playnite.</sys:String>
+    <sys:String x:Key="LOCSettingsForceDownloadPlaynite">Force sync Play Time for supported libraries</sys:String>
+    <sys:String x:Key="LOCSettingsForceDownloadPlayniteTooltip">Always download recorded Play Time from remote library during library sync, ignoring time recorded by Playnite. GOG, Origin and Steam libraries are supported when the user is authenticated.</sys:String>

```

Comment:
I don't think that play time should be capitalized. Also I don't like listing specific libraries here, because if we add support for other in future, I'm pretty sure we will forget to update this.

Reply:
I'd make sure to update the string and send a PR as soon as a new library gets play time support. I think there should be a mention somewhere within Playnite of what libraries support this function but I'll remove it then but tell me if you are ok with it to add it back.